front/components/ideaDetail/IdeaComments.vue

```diff
@@ -1,17 +1,25 @@
 <template>
   <div class="comments-part">
-    <div class="comments-part__header text-center">
-      <strong class="muted">Comments</strong>
-    </div>
-    <v-container ref="scroller" class="comments-part__container">
-      <div v-if="commentList.length" ref="commentsCol" class="">
+    <v-container
+      ref="scroller"
+      class="comments-part__container hide-scrollbar"
+      :class="{ 'fade-bottom': !isMobile }"
```

Comment:
it is using navigator.share which you catch error so basically it will end up into false on ssr but true on mobile so it will lead to dom mismatch error

Reply:
Hm, so then I am wondering now if it makes sense at all then to use isMobile? Seems to be causing more issues than it's worth when we can just use a media query instead.

Reply:
@aldarund do you have a preferred direction on this?

Reply:
If its possible css media query should be used always. Vuetify breakpoint only for things where its not possible to use css media query

Reply:
@f-prime 👆

Reply:
Saw this, will make the fix.

Reply:
The fix has been made. Using media query where possible.